submissions/benjaminKamena/phpDataase/connected.php

```diff
@@ -0,0 +1,31 @@
+<html>
+<?php
+$con = mysqli_connect("localhost", "root", "mysql", "e_commerce_db");
+if (mysqli_connect_errno()){
+	echo "Failed to connect to MySql:" .mysqli_connect_errno();
+} else{
+	//echo "Connected Successfully.";
+	$result = mysqli_query($con,"SELECT * FROM customers");
+	echo "<table border = '1'>
+	<tr>
+	<th>SN</th>
+	<th>fullName</th>
+	<th>EmailAddress</th>
+	<th>CreatedAt</th>
+	<th>Actions</th>
+	</tr>";
+	
+	while($row = mysqli_fetch_array($result)){
```

Comment:
Well done @BenjaminKamena 

But, this implementation shows that each time the page is refreshed, it will always hit the database. Meanwhile, that is not what we want. We want to store the first result we get from making the database query in a cookie. So, as long as that cookie hasn't expired yet, it should take that data stored in that cookie instead of hitting or querying the database again